ci-operator/step-registry/openshift-extended/test/supplementary/openshift-extended-test-supplementary-ref.yaml

```diff
@@ -0,0 +1,51 @@
+ref:
+  as: openshift-extended-test-supplementary
+  from: tests-private
+  grace_period: 10m
+  commands: openshift-extended-test-supplementary-commands.sh
+  cli: latest
+  timeout: 8h0m0s
+  env:
+  - name: TEST_SCENARIOS_SUPPLEMENTARY
+    default: |
+        "25436"
+    documentation: |-
+      Select the test case. It's a RE pattern and is passed to the 'grep -E'
+  - name: TEST_ADDITIONAL_SUPPLEMENTARY
+    default: ""
+    documentation: |
+      It is additional scenario to combine the TEST_SCENARIOS to make final scenarios.
+  - name: TEST_FILTERS_SUPPLEMENTARY
+    default: "~ChkUpgrade&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&"
+    documentation: |
+      It is filters to select case based on some case attribution which is in case title.
+  - name: FILTERS_ADDITIONAL_SUPPLEMENTARY
+    default: ""
+    documentation: |
+      It is additional filters to combine the TEST_FILTERS to make final filters.
```

Comment:
@sunzhaohua2 
change to `It is additional filters to combine the FILTERS_ADDITIONAL_SUPPLEMENTARY to make final filters.`